internal/cli/config/config.go

```diff
@@ -85,6 +85,33 @@ Enter [?] on any option to get help.
 	return nil
 }
 
+func (opts *opts) validateService() error {
+	if opts.Service == config.CloudService {
+		return nil
+	}
+
+	if opts.Service == "gov" {
+		opts.Service = config.CloudGovService
+		return nil
+	}
+
+	if opts.Service == "cloudmanager" || opts.Service == "cm" {
+		opts.Service = config.CloudManagerService
+		return nil
+	}
+
+	if opts.Service == "opsmanager" || opts.Service == "om" {
+		opts.Service = config.OpsManagerService
+		return nil
+	}
+
+	if opts.Service != config.OpsManagerService && opts.Service != config.CloudManagerService && opts.Service != config.CloudGovService {
+		return fmt.Errorf("the service '%s' is not supported. Please run 'mongocli config --help' to see the list of available services", opts.Service)
```

Comment:
```suggestion
		return fmt.Errorf("The '%s' service is not supported. Please run 'mongocli config --help' to see the list of available services.", opts.Service)
```

Reply:
Good practice in go is that the error string should not be capitalized or end with a punctuation mark. I am going to address the comment about `service`